README.md

```diff
@@ -26,3 +26,153 @@ A simple web server can be used to test the build by executing the following com
 
     npm run serve
 
+## Standalone Brouter Server
+
+To run a standalone Brouter HTTPServer that has been containerized, the following steps need to be executed.
+
+### Step 1: Build the docker image
+
+To build the docker image one needs to be sure of the LAT and LNG of the location of interest.  
```

Comment:
```suggestion
The docker image requires two [build arguments](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg), LAT and LNG. These arguments are used to download a segment of the world that will be used by Brouter to provide routing. As of right now, the docker image only downloads one of these segment files because all of the WE area is contained in one slice.  For WE the values are: `N40` and `W85`. Go [here](http://brouter.de/brouter/segments4/) if you'd like to see what other segments are available.
```